packages/config/config/devices/0x027a/zen30.json

```
@@ -343,174 +186,79 @@
 		},
 		{
 			"#": "19",
+			"$import": "~/templates/master_template.json#smart_switch_mode_0-2",
 			"label": "Enable/Disable Load Control for Dimmer",
-			"valueSize": 1,
-			"defaultValue": 1,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Manual control disabled, Z-Wave enabled",
-					"value": 0
-				},
-				{
-					"label": "Manual control enabled, Z-Wave enabled",
-					"value": 1
-				},
-				{
-					"label": "Manual and Z-Wave control disabled",
-					"value": 2
-				}
-			]
+			"defaultValue": 1
 		},
 		{
 			"#": "20",
+			"$import": "~/templates/master_template.json#smart_switch_mode_0-2",
 			"label": "Enable/Disable Load Control for Relay",
-			"valueSize": 1,
-			"defaultValue": 1,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Manual control disabled, Z-Wave enabled",
-					"value": 0
-				},
-				{
-					"label": "Manual control enabled, Z-Wave enabled",
-					"value": 1
-				},
-				{
-					"label": "Manual and Z-Wave control disabled",
-					"value": 2
-				}
-			]
+			"defaultValue": 1
 		},
 		{
 			"#": "21",
-			"label": "Manual Dimming Speed",
-			"description": "How many seconds it takes for the dimmer to go from 0% to 100% brightness when pressing and holding the paddle.",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
+			"$import": "templates/zooz_template.json#local_dimming_speed",
 			"defaultValue": 4
 		},
 		{
 			"#": "23",
+			"$import": "~/templates/master_template.json#base_0-99_nounit",
 			"label": "Default Brightness Level On for Dimmer",
-			"description": "Brightness level for the dimmer to come on to at single tap. Set 0 for last brightness level.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 99,
-			"defaultValue": 0
-		},
-		{
-			"#": "24",
-			"$if": "firmwareVersion >= 1.4",
-			"label": "Dimmer Reporting Behavior When Physical Control Is Disabled",
-			"description": "Enable or disable dimmer reporting on/off status and changing LED indicator state even if physical and Z-Wave control is disabled.",
-			"valueSize": 1,
+			"description": "Brightness level for the dimmer to come on to at single tap.",
 			"defaultValue": 0,
-			"allowManualEntry": false,
+			"unit": "%",
 			"options": [
 				{
-					"label": "Enabled",
+					"label": "Last brightness level",
```

```suggestion
					"label": "Last non-zero brightness",
```
or something like that. Brightness level is a bit redundant.